ts/types/PhoneNumber.ts

```diff
@@ -21,6 +22,12 @@ export function format(
   }
 }
 
+export const format = memoizee(_format, {
+  primitive: true,
+  length: 2,
+  resolvers: [String, String],
```

Comment:
Funny enough, I think we're memoizing on `<phoneNumber>` and `[object Object]` here. And it will break if a user relinks with a different phone number in a different region code.

Reply:
Indeed, I'll fix that.

Reply:
Okay fixed. :)